benchmarks/run_benchmark.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import logging
+import subprocess
+
+import torchvision
+import torch
+import transformers
+from torchvision.prototype.datasets import load as loadpipe
+import torch.nn.functional as F
+from torchvision import transforms
+import time
+import torch.optim as optim
+import torch.profiler
+
+# Relative imports
+from args import arg_parser
+from benchmarks.datasets import prepare_gtsrb_dataset
+from utils import init_fn
+from datasets import prepare_gtsrb_datapipe
+from trainers import train
+from report import create_report
+
+logging.basicConfig(filename='example.log', level=logging.DEBUG)
+
+
+dataset, ispipe, model_name, batch_size, device, num_epochs, num_workers, shuffle, dataloaderv = arg_parser()
+
+if device.startswith("cuda"):
+    nvidiasmi = subprocess.check_output("nvidia-smi", shell=True, text=True)
+    logging.debug(nvidiasmi)
+
+lscpu = subprocess.check_output("lscpu", shell=True, text=True)
+logging.debug(lscpu)
+
+
+if dataloaderv == 1:
+    from torch.utils.data import DataLoader
+elif dataloaderv == 2:
+    from torch.utils.data.dataloader_experimental import DataLoader2 as DataLoader
```

Comment:
Please avoid using this one, perhaps you need to create your own wrapper, which will use DLv2 from torchdata repo and automatically create MultiProcessingReadingService

Reply:
This is up to @NivekT , if he wants to have this update as follow-up or within this PR.

Reply:
@NivekT do you have more context, I'm not sure I follow what change is being requested

Reply:
I think `dataloader_experimental.DataLoader2` is bound to be removed eventually. The actual `DataLoader2` is in `torchdata.dataloader2`.

I have an example in my benchmarking PR, if you're interested https://github.com/pytorch/vision/pull/6196/files#diff-32b42103e815b96c670a0b5f0db055fe63f10fc8776ccbb6aa9b61a6940abba0R207-R211